### DISCUSSION TITLE
Welcome to easy-steam-free-packages Discussions!

### DISCUSSION THREAD

**Discussion by 5x** [5x/easy-steam-free-packages#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community.

**Comment by AppleCanon** [comment]
I have to wait 8 days for this finish :D

**Reply by sscxpp** [reply]
hey i know this is super late but did it work? and how is it?
if u ever see this jus tell me everything about how it worked out for you

edit: do i have to keep my pc on for many hours or days?
